bootstrap.sh

```diff
@@ -1,12 +1,16 @@
 #!/bin/bash
 set -eu
 
+KUBECTL_VERSION=v1.24.15
+HELM_VERSION=v3.12.2
+
 ## Make sure we have Ruby 3 installed directly on the instance
-amazon-linux-extras install -y ruby3.0
+dnf install -y ruby3.2
```

Comment:
I'm concerned about pinning this to a version. I did a little research into this over the weekend and afaict the package repository seems to only support the latest minor version by default. If this is accurate, I believe this would fail once ruby 3.3 comes out (the first preview release rolled out in May, but I do not know how long their preview periods typically last). 

I think we should confirm what the version support in the package manager is, and if it does truly only support the latest minor version the I think we should either leave the install as `dnf install -y ruby` or, if there are concerns about version compatibility, install it manually rather than from the package repository.

Reply:
We use ruby for some pretty basic scripts in the `buildkite-assets` repository, e.g. [here](https://github.com/panorama-ed/buildkite-assets/blob/main/check_command_whitelist.rb) to check the allow-list for commands the Buildkite instance can run.

Reply:
I think it makes more sense to pin to a specific version using some other method. It will save us headache in trying to fix a version compatibility issue that crashes our deployment pipeline.

Reply:
(After discussion, we agreed to go with:)
> yum install -y ruby